repose-aggregator/core/repose-core/src/main/scala/org/openrepose/powerfilter/FilterInitializationException.scala

```diff
@@ -7,24 +7,20 @@
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * 
+ *
  *      http://www.apache.org/licenses/LICENSE-2.0
- * 
+ *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
  */
-package org.openrepose.powerfilter;
+package org.openrepose.powerfilter
 
-public class FilterInitializationException extends Exception {
-    public FilterInitializationException(String message, Throwable t) {
-        super(message, t);
-    }
-
-    public FilterInitializationException(String message) {
-        super(message);
-    }
+class FilterInitializationException(message: String, cause: Throwable = null) extends Exception(message, cause) {
+  def this(message: String) {
+    this(message, null)
```

Comment:
Good point.
Updated.

Reply:
Now I remember why this was needed. There is a single use of this class in `FilterContextFactory` that is a Java class and can't use the defaulting constructor.
I added a comment to this one as well so it can be removed as part of `REP-7231`.

Reply:
Ah, gotcha. Compatibility. Fair enough.